packages/lib-classifier/src/store/SubjectViewerStore.js

```diff
@@ -19,6 +19,15 @@ const SubjectViewer = types
     onZoom: function (type, zoomValue) {},
   }))
 
+  .views(self => ({
+    get interactionMode () {
+      if (self.annotate && !self.move) return 'annotate'
+      if (!self.annotate && self.move) return 'move'
+      
+      return 'annotate'  // Default interaction mode
```

Comment:
Am I being thick, or could this simply be:

```
return (!self.annotate && self.move) ? 'move' : 'annotate'
```

Reply:
No, you're right - the `? :` structure makes more sense. My current structure only makes sense if the default `return` gave something else.